Jenkinsfile

```diff
@@ -301,6 +305,9 @@ pipeline {
                      defaultValue: false,
                      description: 'Run the Functional on Ubuntu 20.04 test stage' +
                                   '  Requires CI_MORE_FUNCTIONAL_PR_TESTS')
+        booleanParam(name: 'CI_medium_large_TEST_SKIP',
```

Comment:
If we want to keep this - which I'm personally on the fence about - a better name would be `CI_FUNCTIONAL_HW_TEST_SKIP`

Reply:
Let's keep it, until we make general cleanup of Jenkinse parameters,